client/human/HumanClientFSM.cpp

```diff
@@ -138,23 +140,27 @@ IntroMenu::~IntroMenu()
 boost::statechart::result IntroMenu::react(const HostSPGameRequested& a) {
     TraceLogger(FSM) << "(HumanClientFSM) IntroMenu.HostSPGameRequested";
     Client().Remove(Client().GetClientUI().GetIntroScreen());
+    Sound::GetSound().PlayBackgroundMusic();
     return transit<WaitingForSPHostAck>();
 }
 
 boost::statechart::result IntroMenu::react(const HostMPGameRequested& a) {
     TraceLogger(FSM) << "(HumanClientFSM) IntroMenu.HostMPGameRequested";
     Client().Remove(Client().GetClientUI().GetIntroScreen());
+    Sound::GetSound().PlayBackgroundMusic();
```

Comment:
I think it makes more sense to put calls to start playing music when constructing an FSM state, rather than when reacting to an event in a state that is being exited. eg. https://github.com/freeorion/freeorion/pull/3358/files#diff-5926c0324401788b2baa5920aff99a5f1d8938f2694222070badba11218400d6

Reply:
I'd also start the background music when actually in the game, not when transiting to the MP lobby.